frontend/__mocks__/mockNotebooks.ts

```diff
@@ -0,0 +1,376 @@
+// api/k8s/apis/kubeflow.org/v1/namespaces/project/notebooks
+
+export const mockNotebooks =  {
+    "apiVersion": "kubeflow.org/v1",
+    "items": [
+        {
+            "apiVersion": "kubeflow.org/v1",
+            "kind": "Notebook",
+            "metadata": {
+                "annotations": {
+                    "notebooks.kubeflow.org/last-activity": "2023-02-14T21:45:14Z",
+                    "notebooks.opendatahub.io/inject-oauth": "true",
+                    "notebooks.opendatahub.io/last-image-selection": "s2i-minimal-notebook:py3.8-v1",
+                    "notebooks.opendatahub.io/last-size-selection": "Small",
+                    "notebooks.opendatahub.io/oauth-logout-url": "http://localhost:4010/projects/project?notebookLogout=workbench",
+                    "opendatahub.io/username": "admin",
+                    "openshift.io/description": "",
+                    "openshift.io/display-name": "workbench"
+                },
+                "creationTimestamp": "2023-02-14T21:44:13Z",
+                "generation": 4,
+                "labels": {
+                    "app": "workbench",
+                    "opendatahub.io/dashboard": "true",
+                    "opendatahub.io/odh-managed": "true",
+                    "opendatahub.io/user": "admin"
+                },
+                "managedFields": [
```

Comment:
I'd really like to see this not in our mocks -- is this generated?

(this applies to all mocks)

Reply:
`managedFields` and any k8s-configuration annotation -- effectively the large blocks of k8s-esk things. They have no value but to bloat the code and make it harder to read the real values / update the values.

Reply:
I will go through the mocks and trim them